services/QuillLMS/config/initializers/rails_admin.rb

```diff
@@ -142,3 +142,13 @@
     config.excluded_models.push(*MODELS_TO_EXCLUDE)
   end
 end
+
+# Monkey patch for a known issue: RailsAdmin tries to parse search strings as JSON
+# https://github.com/railsadminteam/rails_admin/issues/2502
+class RailsAdmin::Config::Fields::Types::Json
+  def parse_value(value)
```

Comment:
Also, looks like this maps to this [Sentry issue](https://quillorg-5s.sentry.io/issues/4475454913/?environment=production&project=11238&query=is%3Aunresolved&referrer=issue-stream&statsPeriod=24h&stream_index=20). Can we write a failing test for this and then fix it?

Reply:
added